templates/partial/_navigation.html

```diff
@@ -20,7 +20,7 @@
       <!-- TO DO - update navigation -->
       <ul class="p-navigation__links" role="menu">
         <li
-          class="p-navigation__link {% if request.path == '/about' %}is-selected{% endif %}"
+          class="p-navigation__link {% if request.path in ['/about', '/about/manifesto', '/about/publishing', '/about/governance', '/about/glossary'] %}is-selected{% endif %}"
```

Comment:
```suggestion
          class="p-navigation__link {% if request.path.startswith('/about') %}is-selected{% endif %}"
```
I think that should work

Reply:
I didn't know this. It works.Thanks